WordPress/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@
     <!-- post view -->
     <string name="post_id">Post</string>
     <string name="upload_failed">Upload failed</string>
+    <string name="draft_uploaded">Draft uploaded</string>
```

Comment:
>Draft uploaded' seems to me to less clearly communicate that it's still a draft, but now it's online

Ahh gotcha now - thanks for explaining and yes feel free to ping for design input :bow: